e2e/ansible/playbooks/hyperconverged/test-k8s-memleak/test-mem.py

```diff
@@ -0,0 +1,31 @@
+#------------------------------------------------------------------------------
+#!/usr/bin/env python
+#description     :Test to verify the memory consumed with sample workloads. 
+#==============================================================================
+
+from __future__ import division
+import subprocess
+import time, os
+list = []
+n = 10
+count = 0
+while count < n:
+    count = count + 1
+    #Obtaining memory consumed using free command from the node where the controller
+    #is scheduled.
+    cmd_used_mem = "sudo free | awk ' /'Mem'/ {print $3}'"
+    out = subprocess.Popen(cmd_used_mem,stdout=subprocess.PIPE, shell=True)
+    used_mem = out.communicate()
+    mem_in_mb = int(used_mem[0])/1024
+    time.sleep(20)
+    list.append(mem_in_mb)
+print list
+# A watermark of 500MB has been set based on benchmark runs with the workload 
+# profile chosen in this test
+# TODO: Identify better mem consumption strategies
+if all(i <= 500 for i in list):
+        print "Pass"
+else:
+        print "Fail"
+
+
```

Comment:
@dargasudarshan , couple of observations:

- While free is an indicator of memory consumption, let us try using pmap which seems to be an indicator of the *type of memory* used. This maybe good checks to have in the testcase. However, it might also be a good idea to parameterize the type of check itself - Having said that, if this is checking the mem issue - it should be ok to implement/track the suggestion as a separate issue.

- While using python/bash scripts might be inevitable in cases and ansible does allow it, it might be a good idea to check if the same can be accomplished via ansible itself. For ex:  Get memory via : 

```
kubectl exec pvc-c71e49dc-273a-11e8-b04f-000c298ff5fc-ctrl-5f659cb984-5x8qx -c pvc-c71e49dc-273a-11e8-b04f-000c298ff5fc-ctrl-con -- pmap -x 1 | awk ''/total'/ {print $3}'
```
...along with a do/until/retry/failed_when block for the check.